core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -90,15 +90,15 @@ trait TypesafeConfigConverters extends ConvertHelpers {
   implicit val configConfigConvert: ConfigConvert[Config] = new ConfigConvert[Config] {
     override def from(config: ConfigValue): Either[ConfigReaderFailures, Config] = config match {
       case co: ConfigObject => Right(co.toConfig)
-      case other => fail(WrongType(other.valueType().toString, "ConfigObject", ConfigValueLocation(config), None))
+      case other => fail(WrongType(other.valueType.toString, ConfigValueType.OBJECT.toString, ConfigValueLocation(config), None))
```

Comment:
I see you started using `ConfigValueType` instances to refer to expected types in `WrongType`. In order to make `WrongType` more type-safe, why don't you make `expectedType` a `List[ConfigValueType]`? That way you make sure the expected types passed are always valid.